CalendarFXView/src/main/java/impl/com/calendarfx/view/DayEntryViewSkin.java

```diff
@@ -64,7 +67,21 @@ public DayEntryViewSkin(DayEntryView view) {
         updateStyles();
     }
 
-    private void updateStyles() {
+    /**
+     * This method registers the given listeners so that the node will be notified about changes of e.g. the entry
+     * which require updates on the UI.
+     */
+    protected void setupUpdateListeners(InvalidationListener updateLabelsListener, InvalidationListener updateStylesListener) {
```

Comment:
I would not pass the listeners. In this class they are already known, and subclasses should not rely on "our" listeners. I have not seen this pattern before and don't want to introduce it in CalendarFX.